requirements.txt

```diff
@@ -6,3 +6,4 @@ pytest>=7.4.0
 scikit-learn>=1.0.1
 scipy>=1.11.1
 setuptools>=58.0.4
+streamlit>=1.34.0
```

Comment:
We might need to install it in some other places, maybe during the Docker image building otherwise streamlit will be missed from the dependencies of our docker container when starting up.

Reply:
I ended up moving this to the Dockerfile so that it will be installed in the container.

Reply:
This is a good solution.